arc/modal/public_api.py

```
@@ -41,10 +42,11 @@
             
         try:
             # Look up the deployed function
-            evaluate_fn = modal.Function.lookup(
-                "arc-production", 
-                "evaluate_single_scenario"
-            )
+            app_name = "arc-production"
+            func_name = "evaluate_single_scenario"
+            
```

Ruff (0.11.9)

Blank line contains whitespace

Remove whitespace from blank line

(W293)